go/test/endtoend/onlineddl/ghost/onlineddl_ghost_test.go

```diff
@@ -159,8 +159,8 @@ func TestMain(m *testing.M) {
 		clusterInstance.VtctldExtraArgs = []string{
 			"-schema_change_dir", schemaChangeDirectory,
 			"-schema_change_controller", "local",
-			"-schema_change_check_interval", "1",
 			"-online_ddl_check_interval", "3s",
+			"-schema_change_check_interval", "1",
```

Comment:
Doesn't matter, but any reason this order of arguments was changed?

Reply:
Bedtime story: it turns out the `onlineddl_ghost` endtoend tests have not been running in a while. There was an exception thrown in this old `TestMain` function that was not caught (in itself something to investigate). What I did now was to copy+paste a brand new `TestMain` function from `onlineddl_vrepl` tests, which works.

Having said that, I'll reorder the arguments just to keep this PR cleaner.